src/services/doppler-api-client.test.ts

```diff
@@ -609,7 +609,7 @@ describe('HttpDopplerApiClient', () => {
           Authorization: `token ${jwtToken}`,
         },
         method: 'GET',
-        url: `/accounts/${accountEmail}/subscribers/${subscriberEmail}/fields/${fieldName}/history`,
+        url: `/accounts/${accountEmail}/subscribers/${subscriberEmail}/permissions-history/${fieldName}`,
```

Comment:
Is it a good practice to get the information using the fieldname instead the field ID? Same with subscriberEmail and acountEmail. 🤔

Reply:
I think that it is when it is possible. For example, in Lists, the name is a kind of description, not an identifier.

But the field name is a kind of identifier, so, I think that we can use it in the URLs, it is nice and self-descriptive.